src/test/java/bio/terra/service/snapshot/SnapshotConnectedTestUtils.java

```diff
@@ -42,7 +42,10 @@
 public class SnapshotConnectedTestUtils {
 
   static SnapshotRequestModel makeSnapshotTestRequest(
-      JsonLoader jsonLoader, DatasetSummaryModel datasetSummaryModel, String resourcePath)
+      JsonLoader jsonLoader,
+      DatasetSummaryModel datasetSummaryModel,
+      String resourcePath,
+      boolean provideProfileId)
```

Comment:
Would it make sense to just have this method take in the profileId and have it be nullable?

Reply:
That works too!